testpmd-container-app/cnfapp/Dockerfile

```
@@ -1,20 +1,33 @@
+FROM docker.io/library/golang:1.21 as build
+
+WORKDIR /scripts
+COPY scripts/webserver.go .
+RUN go mod init webserver.go
+RUN GOOS=linux CGO_ENABLED=0 go build -a -o webserver .
+
 FROM registry.redhat.io/openshift4/dpdk-base-rhel8:v4.6.3-8
 
-MAINTAINER skramaja@redhat.com
+MAINTAINER telcoci@redhat.com
 
 LABEL name="NFV Example CNF Application" \
-      maintainer="skramaja@redhat.com" \
+      maintainer="telcoci@redhat.com" \
       vendor="fredco" \
-      version="v0.2.1" \
-      release="v0.2.1" \
+      version="v0.2.5" \
+      release="v0.2.5" \
       summary="An example CNF for platform valiation" \
       description="An example CNF for platform valiation"
 
 COPY licenses /licenses
 
 USER root
-RUN yum -y update-minimal --disablerepo "*" --enablerepo rhel-8-for-x86_64-baseos-rpms \
-        --enablerepo rhel-8-for-x86_64-appstream-rpms --setopt=tsflags=nodocs \
-        --security --sec-severity=Important --sec-severity=Critical
 
-USER default
+# Copy scripts
+COPY scripts /usr/local/bin
+COPY --from=build /scripts/webserver /usr/local/bin/
+
+# Update permissions
+RUN chmod +x /usr/local/bin/webserver
+RUN chmod +x /usr/local/bin/entrypoint.sh
```

Avoid copying anything that is not needed in the image that is inside the scripts, e.g. the webserver.go

```suggestion
COPY --chmod=550 scripts/entrypoint.sh /usr/local/bin/
COPY --chmod=550 --from=build /scripts/webserver /usr/local/bin/
```